FRCSoftware2024.csv

```diff
@@ -4,31 +4,31 @@ NI-GameTools,ni-frc-2024-game-tools_24.0.1_offline.iso,https://download.ni.com/s
 NI-CompactRIO Driver,ni-compactrio-device-drivers_24.0.0.49260-0+f108_offline.iso,https://download.ni.com/support/nipkg/products/ni-c/ni-compactrio-device-drivers/24.0/offline/ni-compactrio-device-drivers_24.0.0.49260-0+f108_offline.iso,a76eef4d5d81d9f01a1c2ea860ba170a,FALSE
 NI-System Configuration,ni-system-configuration_24.0.0_offline.iso,https://download.ni.com/support/nipkg/products/ni-s/ni-system-configuration/24.0/offline/ni-system-configuration_24.0.0_offline.iso,952156ec99ed9b15fb38be981ee0600f,FALSE
 WPILibInstaller_Windows64,WPILib_Windows-2024.3.1.iso,https://packages.wpilib.workers.dev/installer/v2024.3.1/Win64/WPILib_Windows-2024.3.1.iso,781408b02146fde5a8275f31bd55a23c,FALSE
+WPILib VSCode Windows Archive,VSCode-1.85.1-Windows.zip,https://update.code.visualstudio.com/1.85.1/win32-x64-archive/stable,5621fc9203a0468f9270e2151ac2cdea,TRUE
 Password,2024Password.txt,https://raw.githubusercontent.com/JamieSinn/CSA-USB-Tool/master/2024Password.txt,f7226fa16bbca941473c41beacf2dba4,FALSE
-CTRE Phoenix Windows Installer,CTRE_Phoenix_Framework_v5.30.4.2.exe,https://github.com/CrossTheRoadElec/Phoenix-Releases/releases/download/v5.30.4.2/CTRE_Phoenix_Framework_v5.30.4.2.exe,3a2ae2e2dcece40732f88e4b7baafd17,FALSE
 CTRE Latest Firmware,ctr-device-firmware.zip,https://github.com/CrossTheRoadElec/Phoenix-Releases/raw/master/ctr-device-firmware.zip,00000000000000000,TRUE
-CTRE Tuner X Offline,Tuner_X_v2023.2.2.0_for_Offline_Install.zip,https://github.com/CrossTheRoadElec/Phoenix-Releases/releases/download/v24.2.0/Phoenix-Offline_v24.2.0.exe,4309bff08b61bc6bfb36e946ed1cf6d7,TRUE
-CTRE Linux Offline,CTRE_Phoenix_FRC_Linux_5.30.4.zip,https://github.com/CrossTheRoadElec/Phoenix-Releases/releases/download/v5.30.4.2/CTRE_Phoenix_FRC_Linux_5.30.4.zip,DF58D947801F4FC3A23128E5EE316E4E,TRUE
```

Comment:
I would keep these for now - as the new multiplatform version will make use of them. We want to support options for other OS's.

Reply:
Those are outdated, don't work for 2024, and at least in quick searching, I didn't see that CTRE had a replacement.